projects/gis/src/app/login/login.component.ts

```diff
@@ -50,17 +53,27 @@ export class LoginComponent implements OnInit, AfterViewInit, OnDestroy {
     }
 
     ngOnInit(): void {
-        this.userService
-            .getSessionOrUndefinedStream()
-            .pipe(first())
-            .subscribe((session) => {
-                if (!session || !session.user || session.user.isGuest) {
-                    this.formStatus$.next(FormStatus.LoggedOut);
-                } else {
-                    this.user = session.user;
-                    this.formStatus$.next(FormStatus.LoggedIn);
-                }
-            });
+        // check if OIDC login is enabled
+        this.userService.oidcInit().subscribe(
+            (idr) => {
+                this.oidcUrl = idr.url;
+                this.formStatus$.next(FormStatus.Oidc);
```

Comment:
Why don't we need the subscription here?

Reply:
If the oidcInit gives us the redirect url we can activate the Oidc part of the form. What is there to subscribe?